### DISCUSSION TITLE
Wrong wikidata description showing in iD

### DISCUSSION THREAD

**Discussion by nc011** [openstreetmap/iD#9474] [Q&A] [answered]
Wasn't quite sure where to post this as I'm not sure it's actually a bug with iD editor...but it might be!

An incorrect wikidata description is showing for The Midland Hotel (https://www.openstreetmap.org/way/69284303). It's saying the hotel is a pub (rather than hotel).

However, when I go to the corresponding wikidata entry (https://www.wikidata.org/wiki/Q6842358) the description is correct (it says a hotel).

The description was changed from "pub" to "hotel" back in 2020 (https://www.wikidata.org/w/index.php?title=Q6842358&diff=prev&oldid=1154680004) so why is iD showing the old description?

**Comment by Dimitar5555** [comment, marked as answer]
The UK label was set to "pub". I've just fixed it on Wikidata.
